core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestratorSpec.scala

```diff
@@ -466,140 +443,109 @@ class BillingProjectOrchestratorSpec extends AnyFlatSpec {
+        mock[WorkspaceManagerResourceMonitorRecordDao] // nothing mocked - will fail if called
+      )
     )
+    doReturn(Future.successful()).when(bpo).maybeDeleteGoogleProject(billingProjectName, testContext)
 
     Await.result(bpo.deleteBillingProjectV2(billingProjectName), Duration.Inf)
 
-    // maybeCleanupResources will get called with both lifecycles
-    verify(billingProjectLifecycle, Mockito.times(2))
-      .maybeCleanupResources(billingProjectName, maybeGoogleProject = true, testContext)
-    verify(billingProjectDeletion).finalizeDelete(billingProjectName, testContext, billingProfileExpected = false)
+    verify(bpo).maybeDeleteGoogleProject(billingProjectName, testContext)
+    verify(bpo).maybeDeleteLandingZone(billingProjectName, testContext)
+    verify(billingProjectDeletion).finalizeDelete(billingProjectName, testContext)
   }
 
-  it should "call finalizeDelete when the Azure lifecycle maybeCleanupResources returns a jobId of None" in {
+  it should "call finalizeDelete when the Azure lifecycle cleanupLandingZone returns None (no job started)" in {
     val billingProjectName = RawlsBillingProjectName("fake_billing_account_name")
 
-    val billingProjectLifecycle = mock[BillingProjectLifecycle]
-    when(billingProjectLifecycle.maybeCleanupResources(billingProjectName, maybeGoogleProject = false, testContext))
-      .thenReturn(Future.successful(None))
-
-    val billingProjectDeletion = mock[BillingProjectDeletion]
-    when(billingProjectDeletion.finalizeDelete(billingProjectName, testContext, true)).thenReturn(Future.successful())
-
-    val billingProfileManagerDAO = mock[BillingProfileManagerDAO]
-    val bpo = new BillingProjectOrchestrator(
-      testContext,
-      alwaysGiveAccessSamDao,
-      mock[NotificationDAO],
-      billingProfileManagerDAO,
-      happyBillingRepository(Some(azureBillingProfile), billingProfileManagerDAO),
-      mock[BillingProjectLifecycle](RETURNS_SMART_NULLS), // google
-      billingProjectLifecycle, // azure
-      billingProjectDeletion,
-      mock[MultiCloudWorkspaceConfig],
-      mock[WorkspaceManagerResourceMonitorRecordDao] // nothing mocked - will fail if called
-    )
+    val landingZoneId = UUID.randomUUID()
+    val billingRepository = happyBillingRepository(Some(landingZoneId))
 
-    Await.result(bpo.deleteBillingProjectV2(billingProjectName), Duration.Inf)
+    val azureBillingProjectLifecycle = mock[AzureBillingProjectLifecycle]
+    when(azureBillingProjectLifecycle.cleanupLandingZone(landingZoneId, testContext)).thenReturn(None)
 
-    // Will only call Azure maybeCleanupResources because we know cloud context is Azure
-    verify(billingProjectLifecycle).maybeCleanupResources(billingProjectName, maybeGoogleProject = false, testContext)
-    verify(billingProjectDeletion).finalizeDelete(billingProjectName, testContext, true)
-  }
-
-  it should "call the Azure lifecycle to initiate delete of an Azure project when maybeCleanupResources returns a jobId" in {
```

Comment:
Folded into "create a job to delete the Azure project and set the status of the billing project to Deleting if cleanupLandingZone returns returns a job report".